bindings/java/iota-wallet-java/lib/native/src/lib.rs

```diff
@@ -104,39 +135,30 @@ pub extern "system" fn Java_org_iota_api_NativeApi_sendMessage(
     _class: JClass,
     command: JString,
 ) -> jstring {
-    if env.exception_check().unwrap() {
-        return std::ptr::null_mut();
-    }
+    env_assert!(env, std::ptr::null_mut());
 
-    match MESSAGE_HANDLER.lock() {
-        Ok(message_handler_store) => match message_handler_store.as_ref() {
-            Some(message_handler) => {
-                let command: String = env.get_string(command).expect("Couldn't get java string!").into();
+    let message = jni_from_json_make!(env, command, Message, std::ptr::null_mut());
```

Comment:
Sometimes we pass `()` and sometimes we pass `std::ptr::null_mut()`, I unfortunately don't understand fully what's going on?